.env-40

```diff
@@ -0,0 +1,5 @@
+# docker images
+MW_VERSION?=1.40
+PHP_VERSION?=8.1
+DB_TYPE?=mysql
+DB_IMAGE?="mariadb:11.2"
```

Comment:
What is using these files? And do they have to be in the project root?

Reply:
These are placeholders that can be renamed / symlinked to .env and are then picked up by the "make ci" call. This makes it easier during development to switch between different "targets". But technically, not required. Maybe defining those different targets inside the Makefile would be smarter